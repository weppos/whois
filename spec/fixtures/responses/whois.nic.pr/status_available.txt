%
% NIC.PR WHOIS server.
%
% Disclaimer
% The data in NIC.PR's WHOIS database is provided to you by NIC.PR
% for information purposes only, that is to assist you in obtaining
% information about or related to a domain name registration record.
% NIC.PR makes this information available "as is", and does not 
% guarantee its accuracy. By submitting a WHOIS query, you agree that
% you will use this data only for lawful purposes and that, under no
% circumstances will you use this data to: (1) allow, enable, or 
% otherwise support the transmission of mass unsolicited, commercial 
% advertising or solicitations via direct mail, electronic mail, 
% including spam or by telephone; or (2) enable high volume, 
% automated, electronic processes or robotic, including mining this 
% data for your own personal or commercial purposes that apply to 
% NIC.PR (or its systems). The compilation, repackaging, dissemination
% or other use of this data is expressly prohibited without the prior
% written consent of NIC.PR. NIC.PR reserves the right to modify 
% these terms at any time. By sub-mitting this query, you agree to 
% abide by these terms.

The domain u34jedzcq.pr is not registered.


