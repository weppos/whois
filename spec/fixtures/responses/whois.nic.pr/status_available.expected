#domain
  %s == "u34jedzcq.pr"

#domain_id
  %s %ERROR{AttributeNotSupported}


#status
  %s == :available

#available?
  %s be_truthy

#registered?
  %s be_falsey


#created_on
  %s == nil

#updated_on
  %s %ERROR{AttributeNotSupported}

#expires_on
  %s == nil


#registrar
  %s %ERROR{AttributeNotSupported}

#registrant_contacts
  %s %ERROR{AttributeNotSupported}

#admin_contacts
  %s %ERROR{AttributeNotSupported}

#technical_contacts
  %s %ERROR{AttributeNotSupported}


#nameservers
  %s %CLASS{array}
  %s == []
